docs/fsharp/what-is-fsharp.md

```diff
@@ -0,0 +1,120 @@
+---
+title: What is F#
+description: Learn about what the F# programming language is and what programming with it is like.
```

Comment:
I think this is a good description. But, you may want to expand it for SEO and the search display results for more traffic. Perhaps add a sentence that includes words like "functional", "immutable" or others that might be key search terms.